iocore/net/quic/QUICCongestionController.cc

```diff
@@ -86,7 +86,7 @@ QUICCongestionController::on_packet_acked(const PacketInfo &acked_packet)
 }
 
 void
-QUICCongestionController::on_packets_lost(const std::map<QUICPacketNumber, const PacketInfo *> &lost_packets)
+QUICCongestionController::on_packets_lost(const std::map<QUICPacketNumber, PacketInfo *> &lost_packets)
```

Comment:
Why do we have to remove `const`?

Reply:
Hmm, ok. I'm not sure on_frame_xxx really needs non-const frame but let's think about it when we implement actual functions.